api/src/main/java/org/commonjava/maven/galley/io/SpecialPathConstants.java

```diff
@@ -56,20 +52,36 @@
 
         sp.add( pi );
 
-        pi = SpecialPathInfo.from( new FilePatternMatcher( "maven-metadata\\.xml(\\.md5|\\.sha[\\d]+)?$" ) )
+        pi = SpecialPathInfo.from( new FilePatternMatcher( "maven-metadata\\.xml$" ) )
                             .setMergable( true )
                             .setMetadata( true )
                             .build();
 
         sp.add( pi );
 
-        pi = SpecialPathInfo.from( new FilePatternMatcher( "archetype-catalog\\.xml(\\.md5|\\.sha[\\d]+)?$" ) )
+        pi = SpecialPathInfo.from( new FilePatternMatcher( "maven-metadata\\.xml(\\.md5|\\.sha[\\d]+)$" ) )
+                .setDecoratable( false )
+                .setMergable( true )
+                .setMetadata( true )
+                .build();
```

Comment:
I just meant that we wouldn't actually merge these files...we'd generate a new one for the merged metadata file they are derived from. I'm not sure it matters in practice...but it's not straightforward to create one of these for a group if it's the first thing you request. You have to generate the metadata file it's derived from, then generate it.